build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -786,6 +792,14 @@ message ExecutedActionMetadata {
 
 // An ActionResult represents the result of an
 // [Action][build.bazel.remote.execution.v2.Action] being run.
+//
+// It is advised that at least one non-default field value is set, to
+// differentiate the serialized value from no data. Suitable fields include
+// `ActionResult.execution_metadata.Worker` which could be set to the IP
+// address or hostname of the client that created the ActionResult, any other
+// fields of `ActionResult.execution_metadata`, or the null Digest value
+// for either `ActionResult.stdout_digest.hash` or
+// `ActionrResult.stderr_digest.hash` if all other fields are default values.
```

Comment:
Given the discussion around empty blob digests in #134, I would rather not suggest using the null/empty blob digest values for this purpose.

Reply:
Removed.